inventory/pkg/client/client.go

```diff
@@ -63,6 +65,21 @@ const (
 	InvCacheStaleTimeoutOffsetDescription = "Parameter to set the timeout offset percentage for the Inventory UUID cache"
 )
 
+const (
+	defaultHeatbeatBackoffRetries  = 3
+	defaultHeatbeatBackoffInterval = time.Second * 1
+	defaultHeartbeatInterval       = time.Second * 30
+)
+
+var (
+	heatbeatBackoffRetries = flag.Uint64(
+		"heatbeatBackoffRetries", defaultHeatbeatBackoffRetries, "The number of retries the heartbeat backoff will attempt")
+	heatbeatBackoffInterval = flag.Duration(
+		"heatbeatBackoffInterval", defaultHeatbeatBackoffInterval, "The interval between heartbeat backoff retries")
+	heartbeatInterval = flag.Duration(
+		"heartbeatInterval", defaultHeartbeatInterval, "The interval between heartbeats")
+)
```

Comment:
We don't define flags here actually, but on the components using the client. Should we use the same approach we have for other parameters?